modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectDefinitionModelDocumentContributor.java

```diff
@@ -32,6 +32,7 @@ public void contribute(
 		document.addKeyword(
 			"objectFolderExternalReferenceCode",
 			objectDefinition.getObjectFolderExternalReferenceCode(), true);
+		document.addKeyword(Field.STATUS, objectDefinition.getStatus());
```

Comment:
@sergiojimcos can you say a little bit more about why do we need to add the status as keyword to the ObjectDefinitionModelDocumentContributor?